eos/form-factors/analytic-b-to-gamma-qcdf.cc

```diff
@@ -0,0 +1,187 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2022 Danny van Dyk
+ * Copyright (c) 2022 Philip Lüghausen
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <eos/form-factors/analytic-b-to-gamma-qcdf.hh>
+#include <eos/utils/exception.hh>
+
+#include <numeric>
+
+namespace eos
+{
+    AnalyticFormFactorBToGammaQCDF::AnalyticFormFactorBToGammaQCDF(const Parameters & p, const Options & o):
+        omega_0(   p[par_qname("omega_0")],    *this),
+        blcdas(BMesonLCDAs::make("flvd2022", p, o))
+    {
+        // Verify once that BMesonLCDAs is compatible with this implementation
+        Weights weights;
+        auto [phi_plus_coeff, phi_plus_coeff_end] = blcdas->coefficient_range(1.0);
```

Comment:
Suggestion regarding the naming:
```c++
auto [coeff_begin, coeff_end] = blcdas->coefficient_range(1.0);
```
would be more consistent. Ditto below, where you usually have the ``_end``, but not the ``_begin``.
``coeff_begin`` is fine, the ``phi_plus`` is not needed here I think.